website/docs/r/workflow.html.markdown

```diff
@@ -8,53 +8,29 @@ Create and manage a workflow in New Relic.
 
 # Resource: newrelic\_workflow
 
-Use this resource to create and manage New Relic workflow.
+Use this resource to create and manage New Relic workflows.
 
 ## Example Usage
 
 ##### Workflow
 ```hcl
 resource "newrelic_workflow" "foo" {
   name = "workflow-example"
-  account_id = 12345678
-  enrichments_enabled = true
-  destinations_enabled = true
-  enabled = true
   muting_rules_handling = "NOTIFY_ALL_ISSUES"
 
-  enrichments {
-    nrql {
-      name = "Log"
-      configuration {
-        query = "SELECT count(*) FROM Log WHERE message like '%error%' since 10 minutes ago"
-      }
-    }
-
-    nrql {
-      name = "Metric"
-      configuration {
-        query = "SELECT count(*) FROM Metric WHERE metricName = 'myMetric'"
-      }
-    }
-  }
-
   issues_filter {
     name = "filter-name"
     type = "FILTER"
 
     predicate {
-      attribute = "accumulations.sources"
+      attribute = "tag.team"
```

Comment:
i think we use tags in plural and not singular.

Reply:
I also thought so, but it is actually singular. Also, I forgot to add the required `accumulations.` prefix (will do)